app/assets/javascripts/miq_tree.js

```diff
@@ -394,21 +420,21 @@ function miqSquashToggle(treeName) {
 
 function miqTreeEventSafeEval(func) {
   var whitelist = [
-    'miqOnCheckCUFilters',
-    'miqOnCheckGenealogy',
+    'miqOnCheckCUFilters', // Settings -> Region -> CU Collection tab
+    'miqOnCheckGenealogy', // Compute -> Ingrastructure -> VMs -> Select one vm and click on genealogy
     'miqOnCheckGeneric',
-    'miqOnClickMenuRoles',
-    'miqOnCheckProtect',
-    'miqOnCheckSections',
+    'miqOnClickMenuRoles', // Settings -> Access Controll -> Roles/Edit Roles // Seems like not doing anything
+    'miqOnCheckProtect', // Compute -> Infra -> VMs -> Select VM -> Policy button -> edit
+    'miqOnCheckSections', // Compute -> Infra -> VMs -> Compare VM's
     'miqOnCheckUserFilters',
```

Comment:
Did you cover the affected trees? TreeBuilderBelongsToHac and TreeBuilderBelongsToVat

Reply:
Sorry, I don't understand, cover with what?

Reply:
Does these trees work? Are they covered by your changes? I don't see a path to them here, that's why I'm asking.

Reply:
Yes, they are all covered.